specification/security/resource-manager/common-types/v2/types.json

```diff
@@ -0,0 +1,221 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.0",
+    "title": "Common types"
```

Comment:
`common-types` is a magic path being excluded - otherwise it would've been placed in the same `common` folder as v1

Reply:
Why aren't you using the shared common types at https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v2/types.json?

Reply:
I'd wish it was that easy...

autorest generation fails when trying to do that.
here's an example for only reverting `ServerVulnerabilityAssessmentsSetting.json` common types to the ARM shared types:

```
FATAL: swagger-document/compose - FAILED
FATAL: Error: '$.parameters.SubscriptionIdParameter.description' has incompatible values (---
Azure subscription ID
, ---
The ID of the target subscription. The value must be an UUID.
).
Process() cancelled due to exception : '$.parameters.SubscriptionIdParameter.description' has incompatible values (---
Azure subscription ID
, ---
The ID of the target subscription. The value must be an UUID.
).
  Error: '$.parameters.SubscriptionIdParameter.description' has incompatible values (---
Azure subscription ID
, ---
The ID of the target subscription. The value must be an UUID.
).
```

the problem is that there's a mixture between old and new types sharing the same names, but having different descriptions - the 'v2' types are using the 'v5' models but with our old descriptions to avoid this discrepancy.